tests/unit/article.test.js

```diff
@@ -0,0 +1,53 @@
+import 'chai/register-should';
+import { expect } from 'chai';
+import { validateArticle } from '../../src/utils/article';
+
+const ARTICLE = {
```

Comment:
Let us create a folder for our mock data.

Reply:
@Temmyogunbo Noted